<launch>
    <!-- Throttle the voxel grid that is being published for rviz -->
    <node ns="move_base_node/local_costmap" name="voxel_grid_throttle" pkg="topic_tools" type="throttle" args="messages voxel_grid 3.0 voxel_grid_throttled" />
    
    <!-- for moving -->
    <node pkg="move_base" type="move_base" respawn="false" name="move_base_node" output="screen">
        <remap from="odom" to="odom" />
        <remap from="cmd_vel" to="cmd_vel" />
        
    
        <!-- Load global navigation specific parameters -->
        <rosparam file="$(find auction)/launch/config/move_base_params.yaml" command="load" />
        <rosparam file="$(find auction)/launch/config/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find auction)/launch/config/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find auction)/launch/config/base_local_planner_params.yaml" command="load" />
    </node>
</launch>
